cranelift-codegen/src/ir/function.rs

```diff
@@ -110,6 +110,7 @@ impl Function {
         self.encodings.clear();
         self.locations.clear();
         self.offsets.clear();
+        self.jt_offsets.clear();
```

Comment:
Wow, good catch. Yes, this is one of the downsides of the current scheme of clearing data structures between functions :-/.

Reply:
Maybe add a macro to automatically create the `clear` functions from the struct def?